pkg/cmd/oauth-server/cmd.go

```diff
@@ -63,8 +63,14 @@ func NewOsinServerCommand(out, errout io.Writer, stopCh <-chan struct{}) *cobra.
 	options.Audit.AddFlags(flags)
 
 	flags.StringVar(&options.ConfigFile, "config", "", "Location of the osin configuration file to run from.")
-	cmd.MarkFlagFilename("config", "yaml", "yml")
-	cmd.MarkFlagRequired("config")
+	if err := cmd.MarkFlagFilename("config", "yaml", "yml"); err != nil {
+		fmt.Fprintf(errout, "read config option: %v", err)
+		os.Exit(254)
```

Comment:
No `os.Exit` in library functions! The only acceptable `os.Exit` here is the one in the command's `Run` callback.

Reply:
> os.Exit(255)

https://github.com/openshift/oauth-server/pull/98/files#diff-520d203d74db4a7e2864c3aacc58dc6f70d6f802fda4245334ab0ecd71244d89R53

> klog.Fatal(err) // => os.Exit(255)

https://github.com/openshift/oauth-server/pull/98/files#diff-520d203d74db4a7e2864c3aacc58dc6f70d6f802fda4245334ab0ecd71244d89R41

Could you elaborate, why this is fine here?